packages/@aws-cdk/assets-docker/test/test.image-asset.ts

```diff
@@ -61,22 +61,13 @@ export = {
                   ":",
                   { "Ref": "AWS::AccountId" },
                   ":repository/",
-                  {
-                      "Fn::GetAtt": [
-                        "ImageAdoptRepositoryE1E84E35",
-                        "RepositoryName"
-                      ]
-                  }
+                  { "Fn::GetAtt": [ "ImageAdoptRepositoryE1E84E35", "RepositoryName" ] }
                 ]
               ]
             }
           },
           {
-            "Action": [
```

Comment:
Does it make sense that we lost these permissions?

Reply:
Yes, the user pulling from ECR doesn't need to create logs.